db_stress_tool/db_stress_test_base.cc

```diff
@@ -265,6 +265,12 @@ bool StressTest::BuildOptionsTable() {
                         std::vector<std::string>{"0", "1M", "4M"});
     options_tbl.emplace("blob_file_starting_level",
                         std::vector<std::string>{"0", "1", "2"});
+    options_tbl.emplace("use_blob_cache",
+                        std::vector<std::string>{"false", "true"});
+    options_tbl.emplace("use_shared_block_and_blob_cache",
+                        std::vector<std::string>{"false", "true"});
+    options_tbl.emplace("blob_cache_size",
+                        std::vector<std::string>{"8M", "16M", "64M"});
```

Comment:
The blob cache is not dynamically configurable (it's an "immutable" option), right? If so, these are not needed and I would expect the `SetOptions` call to fail.